src/services/codefixes/fixMissingTypeAnnotationOnExports.ts

```diff
@@ -887,7 +886,7 @@ function withContext<T>(
             type = widenedType;
         }
 
-        if (isParameter(node) && emitResolver.requiresAddingImplicitUndefined(node)) {
+        if (isParameter(node) && typeChecker.getEmitResolver().requiresAddingImplicitUndefined(node)) {

```

Comment:
It turns out the emit resolver is needed _only_ for `requiresAddingImplicitUndefined`. Can we just export this from the checker directly and avoid the resolver entirely? That should save a bunch more time.

Reply:
Right, that seems like a clear improvement.  Done.